File: usermanagement-service/src/main/java/com/michibaum/usermanagement_service/UsermanagementController.java
package com.michibaum.usermanagement_service;

import com.michibaum.usermanagement_library.LoginDto;
import com.michibaum.usermanagement_library.UserManagementEndpoints;
import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequiredArgsConstructor
public class UsermanagementController implements UserManagementEndpoints {


    @Override
    public Boolean checkPassword(LoginDto loginDto) {
        return true;
    }
}

File: usermanagement-service/src/main/java/com/michibaum/usermanagement_service/UsermanagementController.kt
package com.michibaum.usermanagement_service

import org.springframework.web.bind.annotation.RestController
import lombok.RequiredArgsConstructor
import com.michibaum.usermanagement_library.UserManagementEndpoints
import com.michibaum.usermanagement_library.LoginDto

@RestController
@RequiredArgsConstructor
class UsermanagementController : UserManagementEndpoints {

    override fun checkPassword(loginDto: LoginDto): Boolean {
        return true
    }
}